code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -186,7 +188,11 @@
 	M.regenerate_icons()
 	M.update_body()
 
-/datum/game_mode/proc/prepare_syndicate_leader(datum/mind/synd_mind, nuke_code)
+	if(the_bomb)
+		synd_mind.store_memory("<B>Syndicate [the_bomb.name] Code</B>: [the_bomb.r_code]", 0, 0)
+		to_chat(synd_mind.current, "The [the_bomb.name] code is: <B>[the_bomb.r_code]</B>")
+
+/datum/game_mode/proc/prepare_syndicate_leader(datum/mind/synd_mind, var/obj/machinery/nuclearbomb/syndicate/the_bomb)
```

Comment:
Use of `var/` in proc args fails CI.

Dont do this.

Reply:
Whoops. My bad!